src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai.mdx

```diff
@@ -113,48 +110,139 @@ To chat with New Relic AI about log errors:
 3. Search for `errors`.
 2. Click on any resulting log lines and look for the <DoNotTranslate>**Explain this error**</DoNotTranslate> button at the top of the log details panel. Our AI will provide a summary of the error. 
 
-You can take your investigation further by telling our AI about your stack and asking follow up questions. For example, you can ask: 
+You can take your investigation further by telling our AI about your stack and asking follow-up questions. For example, you can ask: 
 
-* How often is this error happening? Meaning, what's the error rate? Is it 1% or 50%?
-* What hosts are throwing these errors? I would like to know is it across all of my hosts or is it just affecting one or two of them?
+```
+How often does this error occur? What's the error rate? Is it 1% or 50%?
+```
+```
+What hosts are throwing these errors? Is it across all of my hosts, or is it just affecting one or two of them?
+```
 
 ### Analyze stack traces in errors inbox [#stack-traces]
 
 Using our AI to analyze stack traces will reduce context switching and help speed up triage and analysis. 
 
-<img
-    title="errors inbox ask ai "
-    alt="A screenshot that shows how to chat with AI in errors inbox"
-    src={aiAskAiinErrorsInbox}
-/>
-
-<figcaption>
-    To chat with AI in errors inbox, go to <DoNotTranslate>**[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities)**</DoNotTranslate> > <DoNotTranslate>**Errors inbox**</DoNotTranslate> > and select a workload. 
-</figcaption>
-
 To investigate stack traces in errors inbox with New Relic AI:
 
 1. Go to <DoNotTranslate>**[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities)**</DoNotTranslate> > <DoNotTranslate>**Errors inbox**</DoNotTranslate>.
 2. Select a workload.
-2. Look for the <DoNotTranslate>**Ask AI**</DoNotTranslate> button at the top of the log details panel. Our AI will provide a summary of the error. 
+3. Look for the <DoNotTranslate>**Explain this error**</DoNotTranslate> button embedded in the Stack trace  panel. Our AI will provide a summary of the error. 
+
+You can dig deeper by asking follow-up questions. For example, you can say: 
+
+```
+Hi, I have an Azure environment and am having issues with Redis being periodically unavailable. Can you tell me more about this error? How can I diagnose this issue? 
+
+```
+### Fill alerting gaps [#coverage-gaps]
+
+As your team and systems evolve, staying ahead of potential issues is crucial. Alerts are your early warning system, but keeping them up-to-date can be challenging. NRAI can help identify areas where you might be missing critical notifications.
```

Comment:
```suggestion
As your team and systems evolve, staying ahead of potential issues is crucial. Alerts are your early warning system, but keeping them up to date can be challenging. New Relic AI can help identify areas where you might be missing critical notifications.
```

Be consistent about calling it fully "New Relic AI" or just "Our AI" after the first reference